src/station_gui.cpp

```diff
@@ -268,8 +276,9 @@ class CompanyStationsWindow : public Window
 						}
 					}
 					/* stations without waiting cargo */
```

Comment:
Several comments in this function need updating to reflect the new strategy of checking the cargo rating instead of the amount of waiting cargo.